packages/react-i18n/CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 <!-- ## [Unreleased] - -->
 
+## [0.11.5] - 2019-03-22
+
+### Added
+
+- Expose `currencyDecimalPlaces` and `DEFAULT_DECIMAL_PLACES` on `react-i18n` [#590](https://github.com/Shopify/quilt/pull/590)
```

Comment:
Nit: `react-i18n` => `@shopify/react-i18n`

Reply:
Actually, this is the changelog for that package, so i'd drop it completely.

Reply:
Sounds good.